.github/workflows/testing.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.14
+          go-version: 1.15
```

Comment:
Is there some yaml way to use a variable for this?

If not, that's fine, but it would be nice to only have to change this and the 2 legacy version numbers whenever there is a Go release.

Reply:
This is driving me nuts. The substitution seemed to work on the vet job, and then I tried it on all the test jobs. Now, they are not running at all. Earlier when I was trying out stuff, at least they used to run and throw some error. Now they are not at all running. I even tried closing and reopening the PR.

Reply:
Looks like it is not possible to do the variable substitution throughout the file. It will work in the `vet-proto` job because we use the variable in the `with` clause, but we wont be able to use it in the matrix.

This is from GA docs:
```
You can only use the `env` context in the value of the `with` and `name` keys, or in a step's if conditional.
```

Reply:
This PR is now back to the state where it was originally. None of the options I tried seem to work :(